### PR TITLE
Doc 11143:  Fixes broken asciisoc syntax, remove s&c content 

### PR DESCRIPTION
**Pull request by franrob-projects**
- Fixes broken asciisoc syntax
- Remove s&c content

### CODE REVIEW

File: modules/ROOT/pages/data-modeling.adoc [hunk 1]
```diff
@@ -73,19 +73,12 @@ You should change any top-level user properties that have a key with a leading u
 * Renaming them to remove the underscore, or,
 * Wrapping them inside another object with a key that doesn't have a leading underscore.
 
-
-
-
-
-
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:80]: Deleting multiple whitespace lines is fine, but just to note, sometimes people find them useful before a `==` to help break up the structure.
I think *2* newlines is sufficient in that case though.

(Skim-reading an asciidoc is hard enough as it is.)

Anyway, just a thought to put that into context.

File: modules/ROOT/pages/data-modeling.adoc [hunk 2]
```diff
@@ -127,25 +120,6 @@ Its main purpose is to enable the replicator to determine which data to sync and
 
 Each document change is assigned a unique revision ID. The IDs of past revisions are available. The content of past revisions may be available if the revision was created locally and the database has not yet been compacted.
 
-[[Scopes-and-Collections]]
-== Scopes and Collections
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:131]: I'm not quite sure why you removed this, but I think you mentioned PM was in agreement?
If so, then 👍

**Review by osfameron — APPROVED** [review]
👍

File: modules/ROOT/pages/data-modeling.adoc (hunk 2 above)
Comment (franrob-projects) [reply modules/ROOT/pages/data-modeling.adoc:131]: Yeah I got some feedback that this was not in the right place. So it's all been shifted to 'whats new' 'Confg' and 'Sync'

**Pull request merged**